homekit/controller/controller.py

```diff
@@ -107,6 +151,7 @@ def identify_ble(accessory_mac):
         :raises AccessoryNotFoundError: if the accessory could not be looked up via Bonjour
         :raises AlreadyPairedError: if the accessory is already paired
         """
+        # TODO needs to be implemented (https://github.com/jlusiardi/homekit_python/issues/74)
```

Comment:
This is done on my test branch (with 2 tests no less)

Reply:
Ok perfect, then I will wait until that merge before investing further work